code/modules/hydroponics/grown.dm

```diff
@@ -518,7 +518,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/grown/ghost_chilli
 	seed = /obj/item/seeds/chillighost
-	name = "ghost chili"
+	name = "ghost pepper"

```

Comment:
A chili shouldn't be called a chili, they're chili peppers. Same logic with the bhut jolokia, if you look at the search result numbers for "ghost pepper", "ghost chili pepper", and "ghost chili" you'll see that ghost pepper has the highest number.